.github/actions/clean_up_package_registry/action.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+name: 'Delete old non-release packages from GitHub package registry'
+description: 'Delete older packages set by a minimal level input'
+author: 'The ORT Project Authors'
+
+inputs:
+  registry:
+    description: 'GitHub container registry'
+    default: 'ghcr.io'
+  token:
+    description: 'GitHub token'
+    required: true
+  keep:
+    description: 'Number of packages we keep after latest'
+    required: false
+    default: 3
+  packages:
+    description: 'Name of the packages should be clean'
```

Comment:
Suggestion: "Names of packages to be cleaned up"

Reply:
Done